packages/atomic/src/components/facets-v1/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -328,22 +330,20 @@ export class AtomicCategoryFacet
     );
   }
 
-  private renderSearchResult(facetValue: CategoryFacetSearchResult) {
-    // TODO: render search result
-    return (
-      <li>
-        <div>{facetValue.displayValue}</div>
-        <div>{facetValue.path.join('/')}</div>
-      </li>
-    );
-  }
-
   private renderSearchResults() {
     return (
       <ul part="search-results" class="mt-3">
-        {this.facetState.facetSearch.values.map((value) =>
-          this.renderSearchResult(value)
-        )}
+        {this.facetState.facetSearch.values.map((value) => (
+          <CategoryFacetSearchResult
+            displayValue={value.displayValue}
+            field={this.field}
+            i18n={this.bindings.i18n}
+            searchQuery={this.facetState.facetSearch.query}
+            numberOfResults={value.count}
+            path={value.path}
```

Comment:
Could the `path`, `numberOfResults` and `displayValue` props be replaced with a single `value: CategoryFacetValue` prop? Or is there a benefit of passing them individually?